### DISCUSSION TITLE
How to pass the IMAGE function into cell

### DISCUSSION THREAD

**Discussion by NeroJz** [exceljs/exceljs#2823] [Q&A]
Hi all, I would like to ask is it possible to call IMAGE function in the cell when using the stream mode?

For instance, I would like to have an exported excel displays the image like this:
https://insider.microsoft365.com/en-us/blog/insert-images-in-cells-with-the-image-function-in-excel

Now when I tried to assign cell.value = {formula: 'IMAGE("some_url")'}, the generated excel will prepend @ in front of IMAGE function (e.g. =@IMAGE("some-image-url"). This results the cell could not display the image.

Appreciate anyone can provide any workaround for this.

Thanks

**Comment by thoro23** [comment]
As a workaround you could use:

`cell.value = { formula: '_xlfn.IMAGE("url")' }`

See https://github.com/exceljs/exceljs/issues/2006 for further information.
